kolibri/plugins/learn/viewsets.py

```diff
@@ -216,10 +222,10 @@ def consolidate(self, items, queryset):
                     "closed": None,
                     "started": False,
                 }
-            exam["missing_resource"] = any(
-                question["exercise_id"] not in available_exam_ids
```

Comment:
Is `available_exam_ids` unused now? If so, we should cleanup the line/query above that defines it.

Reply:
It is still used to generate the `contentnode_map`. I noted a distinction in how `exam_node_ids` and `available_exam_ids` are set along with [filter_by_uuids()](https://github.com/learningequality/kolibri/blob/8ca36542344c514059ddc184b54287843d03f9d7/kolibri/core/mixins.py#L150) being a workaround for a SQLITE quirk, so I left that part unchanged to avoid any unintended side effects.

Reply:
Oh yes - nvm! Apparently I forgot my reading glasses today.